<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminLogin</class>
 <widget class="QDialog" name="adminLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="misc.qrc">
    <normaloff>:/unsoed.png</normaloff>:/unsoed.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#adminLogin {
	background-color:  rgb(255, 153, 70);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox {
	border: 1px solid #fcf9f4;
	border-radius: 10px;
	background-color:  rgb(255, 153, 70);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label3 {
	color: #D9FAFF;
	font: 16pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>ADMIN LOGIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>311</width>
      <height>208</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout1">
       <item>
        <widget class="QLabel" name="label1">
         <property name="styleSheet">
          <string notr="true">QLabel#label1 {
	color: #D9FAFF;
	font: 11pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>USERNAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit#usernameEdit {
	color: #000000;
	height: 40px;
	padding-left: 10px;
	font: 14pt &quot;Comic Sans MS&quot;;
	border-radius: 8px;
	border: 1px solid black;
}</string>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="placeholderText">
          <string>username</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout2">
       <item>
        <widget class="QLabel" name="label2">
         <property name="styleSheet">
          <string notr="true">QLabel#label2 {
	color: #D9FAFF;
	font: 11pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>PASSWORD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit#passwordEdit {
	color: #000000;
	height: 40px;
	padding-left: 10px;
	font: 14pt &quot;Comic Sans MS&quot;;
	border-radius: 8px;
	border: 1px solid black;
}</string>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#exitButton {
	border: none;
	border-radius: 5px;
	height: 35px;
	color: #fcf9f4;
	background-color: rgba(234, 118, 89, 230);
	font: 11pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}
QPushButton#exitButton:hover {
	background-color:  rgba(234, 118, 89, 255);
	font: 13pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#loginButton {
	border: none;
	border-radius: 5px;
	height: 35px;
	color: #fcf9f4;
	background-color: rgba(234, 118, 89, 230);
	font: 11pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}
QPushButton#loginButton:hover {
	background-color: rgba(234, 118, 89, 255);
	font: 13pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources>
  <include location="misc.qrc"/>
 </resources>
 <connections/>
</ui>
